tunnel_routes.go

```diff
@@ -170,7 +170,8 @@ func (api *API) DeleteTunnelRoute(ctx context.Context, rc *ResourceContainer, pa
 		return ErrMissingNetwork
 	}
 
-	uri := fmt.Sprintf("/%s/%s/teamnet/routes/network/%s", AccountRouteRoot, rc.Identifier, url.PathEscape(params.Network))
+	// Cannot fully utilize buildURL here because it tries to escape "%" sign from the already escaped "/" sign from Network field.
```

Comment:
```suggestion
	// Cannot fully utilize buildURI here because it tries to escape "%" sign
	// from the already escaped "/" sign from Network field.
```